src/test/java/org/alfresco/api/YamlToJson.java

```diff
@@ -78,6 +81,10 @@ public static void main(String[] args) {
             swagger = parseSwaggerDef(new File(rootPath + SEARCH_DEFINITION), SEARCH_DEFINITION_TITLE);
             Json.mapper().writeValue(new File(rootPath + SEARCH_JSON_DEFINITION), swagger);
 
+          //Search
```

Comment:
Very minor: Maybe change it to `// Search SQL` and fix the indentation?

Reply:
Updated